src/components/page-header/page-header.component.spec.ts

```diff
@@ -87,4 +100,16 @@ describe('Page Header Component', () => {
         expect(pageHeader.textContent).toContain('UX');
     });
 
+    it('should add href to breadcrumbs with a routerLink', () => {
+        const homeBreadcrumb = nativeElement.querySelector('.breadcrumb a');
+        expect(homeBreadcrumb.textContent.trim()).toBe('Home');
+        expect(homeBreadcrumb.hasAttribute('href')).toBe(true);
+    });
+
+    it('should not add href to breadcrumbs without a routerLink', () => {
```

Comment:
The acceptance mentions a new test page, but I think these karma tests achieve the same result and are probably faster than an e2e test anyway. So I'll stick with these unless there's a specific need for an e2e test that I'm not aware of yet